test/specs/orgBrowser.e2e.ts

```diff
@@ -113,60 +110,58 @@ describe('Org Browser', async () => {
   step('Refresh Org Browser and check MyClass is there', async () => {
     utilities.log(`${testSetup.testSuiteSuffixName} - Refresh Apex Classes`);
     // Check MyClass is present under Apex Classes section
-    await utilities.executeQuickPick('View: Show Org Browser', utilities.Duration.seconds(5));
+    await utilities.openOrgBrowser(utilities.Duration.seconds(1));
     const refreshComponentsButton = await (
-      await utilities.findElementByText('div', 'aria-label', 'Apex Classes')
-    ).$('li[title="SFDX: Refresh Components"]');
+      await utilities.findTypeInOrgBrowser('Apex Classes')
+    ).$('a[aria-label="SFDX: Refresh Components"]');
     await refreshComponentsButton.click();
     await utilities.pause(utilities.Duration.seconds(5));
     const refreshTypesButton = await utilities.findElementByText(
-      'li',
-      'title',
+      'a',
+      'aria-label',
       'SFDX: Refresh Types'
     );
     await refreshTypesButton.click();
     await utilities.pause(utilities.Duration.seconds(5));
-    const myClassLabelEl = await utilities.findElementByText('div', 'aria-label', 'MyClass');
+    const myClassLabelEl = await utilities.findTypeInOrgBrowser('MyClass');
     await expect(myClassLabelEl).toBeTruthy();
   });
 
   xstep('Retrieve This Source from Org', async () => {
     utilities.log(`${testSetup.testSuiteSuffixName} - Retrieve This Source from Org`);
-    const myClassLabelEl = await utilities.findElementByText('div', 'aria-label', 'MyClass');
+    const myClassLabelEl = await utilities.findTypeInOrgBrowser('MyClass');
     await myClassLabelEl.click();
     await utilities.pause(utilities.Duration.seconds(2));
-    const retrieveSourceButton = await utilities.findElementByText(
-      'li',
-      'title',
-      'Retrieve This Source from Org'
+    const retrieveSourceButton = await myClassLabelEl.$(
+      '//a[@aria-label="Retrieve This Source from Org"]'
     );
-    utilities.debug(`button 2 ${JSON.stringify(retrieveSourceButton)}`);
-    await retrieveSourceButton.click();
+
+    await browser.elementHover(retrieveSourceButton.elementId);
```

Comment:
FYI that `elementHover` produces this error when I run the E2E test locally:
```
[chrome 124.0.6367.243 mac #0-0] 1) Org Browser Retrieve This Source from Org
[chrome 124.0.6367.243 mac #0-0] unknown command: unknown command: session/2271a149135e0a50b27449d6317a29a7/element/f.EEE8F9C61C76DF8548C27053A71E75AC.d.CD3FE8E0990D4FB67C7FAA8F1DCD5203.e.859/hover
[chrome 124.0.6367.243 mac #0-0] unknown command: unknown command: unknown command: session/2271a149135e0a50b27449d6317a29a7/element/f.EEE8F9C61C76DF8548C27053A71E75AC.d.CD3FE8E0990D4FB67C7FAA8F1DCD5203.e.859/hover
[chrome 124.0.6367.243 mac #0-0]     at async Context.<anonymous> (file:///Users/daphne.yang/Development/salesforcedx-vscode-automation-tests/test/specs/orgBrowser.e2e.ts:139:5)
```

So I think this function exists in WDIO but is not actually supported for local runs.  (I haven't tried on GHA.)

Reply:
What do you mean "local runs?" Its my understanding that wdio works the same wherever it is run.

Reply:
By "local runs" I meant running the E2E test locally on my computer.  In that case, this function should be broken on a GHA run too.